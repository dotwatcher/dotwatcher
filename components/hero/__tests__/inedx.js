import 'jest-styled-components';

import Hero from '../index';
import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';

describe('<Hero />', () => {
	it('renders <Hero />', () => {
		const tree = renderer.create(<Hero title="Hero Title" byline="Hero Byline" />).toJSON()
		expect(tree).toMatchSnapshot()
	});
});
