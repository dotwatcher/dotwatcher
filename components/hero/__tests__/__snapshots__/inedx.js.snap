// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Hero /> renders <Hero /> 1`] = `
.c0 {
  background-color: var(--washed-blue);
  color: var(--black-90);
  padding-left: var(--spacing-large);
  padding-right: var(--spacing-large);
  padding-bottom: var(--spacing-large);
  *zoom: 1;
}

.c1 {
  font-size: 6rem;
  margin-bottom: var(--spacing-none);
}

.c2 {
  font-size: 3rem;
  margin-top: var(--spacing-none);
  max-width: 20em;
}

<header
  className="cf c0"
>
  <h1
    className="c1"
  >
    Hero Title
  </h1>
  <h2
    className="c2"
  >
    Hero Byline
  </h2>
</header>
`;
